examples/gno.land/r/sys/params/unlock_test.gno

```diff
@@ -1,54 +1,63 @@
 package params
 
 import (
+	"std"
 	"testing"
 
-	"gno.land/p/demo/dao"
-	"gno.land/p/demo/simpledao"
 	"gno.land/p/demo/urequire"
-	"gno.land/r/gov/dao/bridge"
-
-	// loads the latest DAO implementation in the bridge.
-	_ "gno.land/r/gov/dao/init"
+	"gno.land/r/gov/dao"
+	"gno.land/r/gov/dao/v3/init"
 )
 
+func init() {
+	c := std.OriginCaller()
+	init.InitWithUsers(c)
+}
+
 func TestProUnlockTransfer(t *testing.T) {
-	govdao := bridge.GovDAO()
-	id := ProposeUnlockTransfer()
-	p, err := govdao.GetPropStore().ProposalByID(id)
+	pr := ProposeUnlockTransferRequest()
+	std.TestSetOriginPkgAddress(std.Address("g1p84dvfh4wrplyxx4zsmy77a8rxcnjup2j7zv5r")) // gno.land/r/gov/dao
+	id := dao.MustCreateProposal(pr)
+	p, err := dao.GetProposal(id)
```

Comment:
Is it technically possible at this point to check that the lock param is changed after executing the proposal? If so it might be good to check it.

Reply:
we can only check if the SetXXX returns an error; but there is no Getter yet, and it's not planned to add them atm.